src/components/SRNEntry.vue

```diff
@@ -55,7 +59,16 @@ export default {
     return {
       userIDList: [{ userID: "" }],
       invalidInputMessage: null,
+      validated: true,
```

Comment:
```suggestion
      doesUserExists: true,
```
Suggestion - 
Two things, for boolean values, it's super helpful to have `is` or `does` in the beginning.
And I think this variable name accurately represents the output of `checkUserExists` method